### PR TITLE
Feature Functionality

### PR DESCRIPTION
**Pull request by IsmailChangezi**
### Description
- Structured the page with Index.html 
- Styled the page with Styles.css
- Added functionality to the page with Main.js
- Fixed Linter errors

### CODE REVIEW

**Review by rustam-tolipov — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — COMMENTED** [review]
Hi @IsmailChangezi 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesnt show contribution from your partner and you dont have a descriptive readme. Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: main.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    lastId = books[books.length - 1].id;
+  }
+  return lastId;
+}
+
+let n = lastId();
+
+function printBooks() {
+  let printed = '';
+  //  eslint-disable-next-line
```
Comment (CollinsTatang) [inline main.js:62]: Please do not disable your linters, 
  
- [ ]  Kindly enable all your disable linters and fix them to work as required.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+     
+    </div>
+    `;
+  }
+  listSection.innerHTML = printed;
+}
+
+printBooks();
+
+//  eslint-disable-next-line
```
Comment (CollinsTatang) [inline main.js:78]: Please do not disable your linters, 
  
- [ ]  Kindly enable all your disable linters and fix them to work as required.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+  storeData();
+}
+/*
+1. add eventlistener to List, add new and contact
+list call displayList()
+add new calls displayAddBooks()
+constact calls displayContact()
+when clicking name displayAll()
+*/
```
Comment (CollinsTatang) [inline main.js]: - [ ] Please remove all commented code from your codebase to make your code clean and easy to maintain.

File: main.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  }
+  listSection.innerHTML = printed;
+}
+
+printBooks();
+
+//  eslint-disable-next-line
+function deleteBook(id) {
+  // let newBooks = [];
+  //  eslint-disable-next-line
```
Comment (CollinsTatang) [inline main.js:81]: Please do not disable your linters, 
  
- [ ]  Kindly enable all your disable linters and fix them to work as required.

File: main.js [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+    this.title = title;
+    this.author = author;
+  }
+
+  Add() {
+    books.push({ id: this.id, title: this.title, author: this.author });
+  }
+
+  Delete() {
+    //  eslint-disable-next-line
```
Comment (CollinsTatang) [inline main.js:34]: Please do not disable your linters, 
  
- [ ]  Kindly enable all your disable linters and fix them to work as required.

**Review by CollinsTatang — COMMENTED** [review]
Hi @IsmailChangezi , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't show a contribution from your partner and you don't have a descriptive readme. Add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: main.js (hunk 5 above)
Comment (Davidosky007) [reply main.js:34]: * Kindly remove all inline linter disabling as mentioned by the previous CR

File: main.js (hunk 1 above)
Comment (Davidosky007) [reply main.js:62]: * Kindly remove all inline linter disabling as mentioned by the previous CR

File: main.js (hunk 2 above)
Comment (Davidosky007) [reply main.js:78]: * Kindly remove all inline linter disabling as mentioned by the previous CR

File: main.js (hunk 4 above)
Comment (Davidosky007) [reply main.js:81]: * Kindly remove all inline linter disabling as mentioned by the previous CR and remove all commented codes

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @IsmailChangezi 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Well written PR description 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**